scripts/build-packages-dist.ts

```diff
@@ -6,9 +6,10 @@
  * distribution folder within the project.
  */
 
-const {execSync} = require('child_process');
-const {join} = require('path');
-const {chmod, cp, mkdir, rm, set, test} = require('shelljs');
+import {execSync} from 'child_process';
```

Comment:
There are a number of types missing for parameters in the file.

Additionally, it looks like the file permissions changed from `755` -> `644`, since this is now ts instead of js this may be intentional/better, but I wanted to highlight it.